Documentation/network/kubernetes/compatibility-table.rst

```diff
@@ -28,6 +28,8 @@
 +-----------------+----------------+
 | v1.11.8         | 1.24.4         |
 +-----------------+----------------+
+| v1.11.9         | 1.24.4         |
```

Comment:
These are just updating the compatibility matrix with older versions that were released since the last RC?

Reply:
Yeah, the idea is that the docs are relatively up to date to describe which Cilium versions use which CRD schema versions for CNP/CCNP. There's a script that generates this table for us.